src/FloatConfig.res

```diff
@@ -0,0 +1,73 @@
+type erc20 = {
+  address: string,
+  symbol: string,
+  iconUrl: option<string>,
+}
+
+type contract = {
+  address: string,
+  description: option<string>,
+  codeUrl: option<string>,
+  abiUrl: option<string>,
+}
+
+type contracts = {
+  longShort: contract,
+  staker: contract,
+  stakeBalance: contract,
+  tokenFactory: contract,
+  floatToken: contract,
+  floatCapital_v0: contract,
+  treasury_v0: contract,
+  gems: contract,
+  gemCollectorNft: contract,
+}
+
+type paymentToken = {
+  address: string,
+  isUSDBased: bool,
+  decimals: int,
+  symbol: string,
+  iconUrl: option<string>,
+}
+
+type oracle = {
```

Comment:
Might want to store the oracle address here too

Reply:
Cool will add that